docker/Dockerfile.hardware

```
@@ -28,17 +27,17 @@ RUN apt-get update  && \
     source /opt/ros/$ROS_DISTRO/setup.bash && \
     colcon build --symlink-install --packages-up-to husarion_ugv --cmake-args -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=$BUILD_TEST && \
     # Get version
-    echo $(cat /ros2_ws/src/panther_ros/husarion_ugv/package.xml | grep '<version>' | sed -r 's/.*<version>([0-9]+.[0-9]+.[0-9]+)<\/version>/\1/g') >> /version.txt && \
-    # Size optimalization
+    echo $(cat /ros2_ws/src/husarion_ugv/husarion_ugv/package.xml | grep '<version>' | sed -r 's/.*<version>([0-9]+.[0-9]+.[0-9]+)<\/version>/\1/g') >> /version.txt && \
+    # Size optimization
     export SUDO_FORCE_REMOVE=yes && \
     apt-get remove -y \
         ros-dev-tools && \
     apt-get autoremove -y && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
-COPY update_config_directory.sh /usr/local/sbin/update_config_directory
-COPY setup_environment.sh /usr/local/sbin/setup_environment
+COPY utils/update_config_directory.sh /usr/local/sbin/update_config_directory
+COPY utils/setup_environment.sh /usr/local/sbin/setup_environment
```

```suggestion
COPY docker/utils/update_config_directory.sh /usr/local/sbin/update_config_directory
COPY docker/utils/setup_environment.sh /usr/local/sbin/setup_environment
```
Talking about right context

---

Thanks for catching that